src/model_signing/_cli.py

```diff
@@ -66,6 +68,15 @@
     help="File paths to ignore when signing or verifying.",
 )
 
+# Decorator for the commonly used option to ignore git related paths
+_ignore_git_paths_option = click.option(
+    "--ignore-git-paths",
+    type=bool,
+    is_flag=True,
+    default=False,
```

Comment:
I think this should be default `True` and since it's a flag users can pass `--ignore-git-paths=False` (or we can also make it support `--no-ignore-git-paths`)